keras/backend/mxnet_backend.py

```diff
@@ -3985,6 +4005,24 @@ def _preprocess_convnd_kernel(kernel):
         kernel = KerasSymbol(mx.sym.transpose(data=kernel.symbol, axes=(3, 2, 0, 1)))
     return kernel
 
+@keras_mxnet_symbol
+def _preprocess_convnd_transpose_kernel(kernel):
+    """
+    Channel and Number of Filter is reversed compare to convolution kernel
+    TODO: add support for conv3d transpose kernel
+    """
+    if len(kernel.shape) > 3:
+        kernel = KerasSymbol(mx.sym.transpose(data=kernel.symbol, axes=(2, 3, 0, 1)))
+    return kernel
+
+
+@keras_mxnet_symbol
```

Comment:
We don't require `@keras_mxnet_symbol` here I guess since we are not dealing with symbol here.